tests/src/Appender/RollingFileAppenderTest.cs

```diff
@@ -240,13 +252,52 @@ public void RollingCombinedWithPreserveExtension()
 			VerifyFileCount(2, true);
 		}
 
+		[Test]
+		public void RollingBySizePreserveExtensionAndWithFolder()
```

Comment:
.. or `RollingBySizeAndPreserveExtensionShouldWorkWithSubFolders` because it is shorter?